src/pynguin/ga/stoppingcondition.py

```diff
@@ -162,6 +162,60 @@ def after_search_finish(self) -> None:
         """Not used."""
 
 
+class MinimumCoverageStoppingCondition(StoppingCondition):
+    """A stopping condition that checks if coverage doesn't change over multiple iterations."""
+
+    def _init_(self, min_coverage: float, min_iterations: int):
+        """Create new MinimumCoverageStoppingCondition.
+
+        Args:
+            min_coverage: the minimum coverage after which the algorithm can stop when reached.
+            min_iterations: the minimum number of iterations the algorithm should check whether the coverage changes.
+        """
+        super()._init_()
+        self._num_iterations = 0 #Count up the iterations
+        self._current_coverage = 0.0 #Store current coverage
+        self._last_coverage = 0.0 #Cache the coverage from last iteration
+        assert min_coverage > 0.0
+        assert min_iterations > 0.0
```

Comment:
You might want to check these criteria *before* calling the constructor of this class.  Furthermore, there is not only a minium reasonable coverage value but also a maximum (`< 1.0`).